util/src/main/java/tc/oc/pgm/util/attribute/Attribute.java

```diff
@@ -0,0 +1,45 @@
+package tc.oc.pgm.util.attribute;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum Attribute {
```

Comment:
Let's not do enum. Either an interface or class should be fine here.

Reply:
Are you sure?
The enum methods are used in XMLParsing to make sure its a valid attribute

```    
    tc.oc.pgm.util.attribute.Attribute attribute = tc.oc.pgm.util.attribute.Attribute.byName(text);
    if (attribute != null) return attribute;

    attribute = tc.oc.pgm.util.attribute.Attribute.byName("generic." + text);
    if (attribute != null) return attribute;
```
And for going through and reseting all the attributes on the player in MatchPlayerImpl::reset